sdk/servicebus/Azure.Messaging.ServiceBus/src/GuidUtilities.cs

```diff
@@ -56,7 +56,11 @@ public static void WriteGuidToBuffer(Guid guid, Span<byte> buffer)
 
         if (BitConverter.IsLittleEndian)
         {
+#if NET6_0_OR_GREATER
+            MemoryMarshal.Write(buffer, in guid);
```

Comment:
See: Warning [CS9191 The ref modifier for argument corresponding to in parameter is equivalent to in. Consider using in instead.](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/ref-modifiers-errors?f1url=%3FappId%3Droslyn%26k%3Dk(CS9191)#reference-variables-require-a-referent)

Reply:
Could we use `in` in the netstandard2.0 target as well and avoid multitargeting here?

Reply:
I need to try this again but iirc I was getting an error in netstandard2.0 that ref was required here haha. I can probably just suppress the warning for this one since the warning itself even says they're equivalent.